<launch>
    <!-- 是否rviz可视化 -->
    <arg name="rviz" default="true" />

    <node pkg="le_calib" type="le_calib_node" name="le_calib_node" output="screen">
        <rosparam command="load" file="$(find le_calib)/config/param.yaml" />
    </node>

    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find le_calib)/rviz/visual.rviz" />
    </group>

</launch>